### DISCUSSION TITLE
Login With Github

### DISCUSSION THREAD

**Discussion by Skillz4Killz** [nestdotland/nest.land#225] [Ideas]
Just tried to use nest.land and realized you guys don't yet support GitHub login. I don't really remember usernames/passwords well if ever. I always forget. I forgot the password to my 1password 🤦  Login with something like Github is essential IMO 🙏

**Comment by t8** [comment]
This is a really neat idea! 

At this point in time, we don't want to have to collect user emails, so we didn't bother setting any sort of oauth up. We also want to function completely independently from source control, so if this were an option, it would need to be completely optional.

I'm not against adding this though, and we really appreciate the feedback.

**Reply by t8** [reply]
I should note that we still would not collect user emails if this were an option. We'd essentially only be using it for usernames.

**Comment by zorbyte** [comment]
We try to be agnostic of other platforms, but I empathise with your situation so I'm quite neutral on this. I think it would be great if we took this a step further and optionally integrated some GitHub features into nest.land if we go ahead with this.
